### DISCUSSION TITLE
Add suffix/prefix (-icon) to number inputs

### DISCUSSION THREAD

**Discussion by Radiergummi** [lowdefy/lowdefy#1019] [Ideas and feature requests]
Just had a situation where I'd wanted to add a unit (km) to a number field (to enter the distance to something) but noticed the suffix option only exists on text fields.

**Comment by Gervwyk** [comment]
Hi @Radiergummi 

I've already implemented the formatter option on the v4 branch, https://github.com/lowdefy/lowdefy/blob/ecdf9246abb9d03df62daeafba3dc186b50c0b8a/packages/plugins/blocks/blocks-antd/src/blocks/NumberInput/NumberInput.js#L54
So that would allow you to add a unit to the number input in the near future :)
It is not implemented in v3 since v3 runs antd@4.4.2 which is < 4.17.0 with formatted support.

But I think we also need to implement the addonBefore and addonAfter options to add support for prefix / suffix icons. Which would be great since v4 implements all [react-icons](https://react-icons.github.io/react-icons/). 

See https://ant.design/components/input-number/#API

**Reply by Radiergummi** [reply]
Ooooh lovely I'm so excited for 4.0 😍
